modyn/trainer_server/internal/trainer/remote_downsamplers/remote_gradnorm_downsampling.py

```diff
@@ -16,9 +17,15 @@ class RemoteGradNormDownsampling(AbstractRemoteDownsamplingStrategy):
     """
 
     def __init__(
-        self, pipeline_id: int, trigger_id: int, batch_size: int, params_from_selector: dict, per_sample_loss: Any
+        self,
```

Comment:
We could also add a reference here to the paper proposing it

Reply:
There's a reference for each downsampler. It is a few lines above for this one. 
```
"""
    Method adapted from
    Not All Samples Are Created Equal: Deep Learning with Importance Sampling (Katharopoulos, Fleuret)
    The main idea is that the norm of the gradient up to the penultimate layer can be used to measure the importance
    of each sample. This is particularly convenient if the loss function is CrossEntropy since a closed form solution
    exists and thus no derivatives are needed (so it's marginally more expensive than computing the loss)
"""
```